app/response_processor.py

```diff
@@ -245,58 +246,33 @@ def send_notification(self):
         self.logger.info("Sending to downstream")
         try:
             self.logger.info("About to publish notification to queue")
-            self.notifications.publish_message(
-                self.tx_id, headers={
-                    'tx_id': self.tx_id
-                })
-        except PublishMessageError as e:
-            self.logger.error("Unable to queue response notification", error=e)
+            self.notifications.publish_message(self.tx_id, headers={'tx_id': self.tx_id})
+        except PublishMessageError:
+            self.logger.exception("Unable to queue response notification")
```

Comment:
Why not logging the `e` now?

Reply:
@necrophonic Assumedly because it has to be a `PublishMessageError`, and that doesn't actually carry anything more useful than knowing what it is.

@insacuri To confirm of course.

Reply:
logger.exception("blah") does the same as logger.error("blah", error=e)

Reply:
Ah nice answer :)